src/explorer/assets/main.html

```diff
@@ -0,0 +1,22 @@
+<!-- Mimi UI -->
+
+<html>
+<header>
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega/3.0.8/vega.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega-lite/2.0.3/vega-lite.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/vega-embed/3.0.0-rc7/vega-embed.js"></script>
```

Comment:
This is fine for the first PR, but @lrennels can you open an issue that in a follow up PR we want to actually just include these files in the repo here and use the local versions, so that things work when one is offline.